lib/pbench/server/api/resources/upload_api.py

```diff
@@ -257,11 +250,7 @@ def _put(self, args: ApiParams, request: Request, context: ApiContext) -> Respon
             )
 
             current_app.logger.info(
-                "PUT uploading {} for user_id {} (username: {}) to {}",
-                filename,
-                user_id,
-                username,
-                tar_full_path,
+                "PUT uploading {} for {} to {}", filename, username, tar_full_path
             )
```

Comment:
Should this be `debug()` rather than `info()`?  (Especially given that Nginx may be providing this visibility?)

Reply:
Good point, and worth thinking about if someone nudges me to make additional changes here; but I'd prefer not to go through another round, and this isn't new with this PR.

We should in general be purging `info` and `warning` log messages from the APIs. I removed redundant repetitions of identical information but it didn't occur to me at the time that it was really *all* redundantly repetitious. 😁